### DISCUSSION TITLE
Excel VBA

### DISCUSSION THREAD

**Discussion by scheur** [community/community#3973] [Other Feature Feedback, Questions, & Ideas]
Having Excel VBA integration with GitHub would be magnificent. 

Somehow personal.xlsb gets lost frequently and isn't syncing across workstations over OneDrive.

**Comment by Levi-Lesches** [comment]
> Somehow personal.xlsb gets lost frequently and isn't syncing across workstations over OneDrive.

Seems like your file isn't being committed to Git in the first place. Check your `.gitignore`?
